core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-layout/edit-ema-layout.component.ts

```diff
@@ -38,11 +46,23 @@ export class EditEmaLayoutComponent implements OnInit, OnDestroy {
     private readonly dotPageLayoutService = inject(DotPageLayoutService);
     private readonly messageService = inject(MessageService);
     private readonly dotMessageService = inject(DotMessageService);
+    readonly #router = inject(Router);
 
     protected readonly uveStore = inject(UVEStore);
 
     protected readonly $layoutProperties = this.uveStore.$layoutProps;
 
+    readonly $handleCanEditLayout = effect(() => {
+        // The only way to enter here directly is by the URL, so we need to redirect the user to the correct page
+        if (this.uveStore.$canEditLayout()) {
+            return;
+        }
+
```

Comment:
Maybe we can render a toast with a friendly message here?
Like "You cannot edit the layout"

Reply:
This is the ticket: https://github.com/dotCMS/core/issues/28819, it wasn't a GD ticket